src/webui/webapplication.cpp

```diff
@@ -306,9 +308,30 @@ void WebApplication::action_command_shutdown()
 void WebApplication::action_command_download()
 {
     CHECK_URI(0);
-    CHECK_PARAMETERS("urls");
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    QString savepath = request().posts["savepath"];
+    QString label = request().posts["label"];
+    QString cookie = request().posts["cookie"];
+    QList<QNetworkCookie> cookies;
+    if (!cookie.isEmpty()) {
+
+        QStringList cookiesStr = cookie.split("; ");
+        foreach (QString cookieStr, cookiesStr)
+        {
```

Comment:
What do you mean exactly? Every file has a different coding style (weird I know), and the webapplication.cpp has
``` 
if () {
```
instead of
```
if () 
{
```
in all other methods.

Unless you mean something else...